### PR TITLE
William/style cleanup

### PR DESCRIPTION
**Pull request by swansontec**
I found these while trying to do the Typescript upgrade.

### CODE REVIEW

File: packages/edge-login-ui-rn/src/components/screens/newAccount/NewAccountReviewScreen.js [hunk 1]
```diff
@@ -34,10 +34,22 @@ class NewAccountReviewScreenComponent extends React.Component<Props> {
...
-              <WarningBox
-                style={styles.warningBox}
-                message={s.strings.warning_message}
-              />
+              <View style={styles.warningBox.container}>
+                <View style={styles.warningBox.bottom}>
+                  <Text style={styles.warningBox.text}>
+                    {s.strings.warning_message}
+                  </Text>
+                </View>
```
Comment (thehobbit85) [inline packages/edge-login-ui-rn/src/components/screens/newAccount/NewAccountReviewScreen.js:42]: Can't you just merged the styles from the View into the Text and have only the Text component?

File: packages/edge-login-ui-rn/src/components/screens/newAccount/NewAccountReviewScreen.js [hunk 2]
```diff
@@ -34,10 +34,22 @@ class NewAccountReviewScreenComponent extends React.Component<Props> {
...
+                <View style={styles.warningBox.top}>
+                  <View style={styles.warningBox.iconWrapBottom}>
+                    <EvilIcons
+                      name="exclamation"
+                      style={styles.warningBox.iconStyle}
+                      size={styles.warningBox.iconSize}
+                    />
+                  </View>
+                </View>
```
Comment (thehobbit85) [inline packages/edge-login-ui-rn/src/components/screens/newAccount/NewAccountReviewScreen.js:51]: Can't you just merged the styles from the View into the EvilIcons and have only the EvilIcons component?

**Review by thehobbit85 — CHANGES_REQUESTED** [review]
Address comments

**Review by thehobbit85 — APPROVED** [review]

File: packages/edge-login-ui-rn/src/components/screens/newAccount/NewAccountReviewScreen.js (hunk 1 above)
Comment (swansontec) [reply packages/edge-login-ui-rn/src/components/screens/newAccount/NewAccountReviewScreen.js:42]: Tried it, but it weirdly broke the layout.

**Pull request merged**